lib/OpenQA/Command.pm

```diff
@@ -52,7 +52,15 @@ sub handle_result {
     my $res     = $tx->res;
     my $is_json = ($res->headers->content_type // '') =~ m!application/json!;
 
-    if (!$options->{quiet} && (my $err = $res->error)) {
+    my $err = $res->error;
+    if ($options->{verbose} && (!$err || $err->{code})) {
```

Comment:
I'm finding the condition a little confusing to read... verbose and no error somehow equals verbose and any error code?

Reply:
The condition excludes connection errors. 4xx and 5xx also result in a `$err` value, but with `$err->{code}` set to a true value.

Reply:
To be precise, connection errors have no `$err->{code}`.

Reply:
But i agree that it is not particularly obvious, so i've added an `$is_connection_error` variable.